client/shared/dev/defaults.bzl

```diff
@@ -0,0 +1,25 @@
+load("@aspect_rules_ts//ts:defs.bzl", _ts_config = "ts_config", _ts_project = "ts_project")
```

Comment:
I think this file (defaults.bzl) should go somewhere common because it will be used I think across every package. 

Either `/defs.bzl` or maybe `/tools/defs.bzl`, or `/tools/ts.bzl`? 🤔